src/Components/Dashboard.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import WarehouseHealthUploaded from './Dashboard_Components/Uploaded_Graphs/WarehouseHealthUploaded.js';
+import WarehouseUsageUploaded from './Dashboard_Components/Uploaded_Graphs/WarehouseUsageUploaded.js';
+import '../main.css'
+
+class Dashboard extends Component {
+
+    render() {
+        if (this.props.healthcheck_data) {
+            return (
+                <div className="mx-auto">
+                    <WarehouseHealthUploaded healthcheck_data={this.props.healthcheck_data}/>
```

Comment:
Please filter the healthcheck data so it only passes down the data required for each chart